File: AndroidLearnProject/app/src/main/code_backup/mcontract/TodoAddContract.java
package com.android.learn.mcontract;

import com.android.learn.base.mmodel.BaseData;

import java.util.Map;

/**
 * Created by gaolei on 2018/6/18.
 */

public class TodoAddContract {

    public interface Presenter {

        void addTodo( Map<String, Object> map);

    }

    public interface View {


        void showAddTodo(BaseData todoData);
    }
}

File: app/src/main/java/com/android/learn/mcontract/TodoAddContract.kt
package com.android.learn.mcontract

import com.android.learn.base.mmodel.BaseData
import com.android.learn.base.mview.BaseView

/**
 * Created by gaolei on 2018/6/18.
 */

class TodoAddContract {

    interface Presenter {

        fun addTodo(map: Map<String, Any>)

    }

    interface View : BaseView {


        fun showAddTodo(todoData: BaseData)
    }
}
